src/write.rs

```diff
@@ -0,0 +1,96 @@
+//! Customize serialization behaviour
+
+use std::{borrow::Cow, fs::File, io::BufWriter, path::Path};
+
+use plist::XmlWriteOptions;
+
+use crate::Error;
+
+/// Options that can be set when writing the UFO to disk.
+#[derive(Debug, Clone)]
+pub struct WriteOptions {
+    // for annoying reasons we store three different representations.
+    pub(crate) indent_str: Cow<'static, str>,
+    xml_opts: XmlWriteOptions,
+    pub(crate) whitespace_char: u8,
+    pub(crate) whitespace_count: usize,
+}
+
+impl Default for WriteOptions {
+    fn default() -> Self {
+        WriteOptions {
+            indent_str: "\t".into(),
+            xml_opts: Default::default(),
+            whitespace_char: b'\t',
```

Comment:
Single tab spacing is the ufonormalizer default? Optimal default for interoperability with other tools?